src/main/java/core/basesyntax/strategy/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.data.FruitTransaction;
+import java.util.Map;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void processWithData(FruitTransaction transaction, Map<String, Integer> data) {
+        String fruit = transaction.getFruit();

```

Comment:
In my opinion, this is a redundant variable

Reply:
I don't agree. It's need for more clear code.